src/StreamMaster.cpp

```diff
@@ -0,0 +1,203 @@
+#include "StreamMaster.h"
+#include "Streamer.h"
+
+FileWriter::StreamMaster::StreamMaster(
```

Comment:
Just a stylistic thing: Usually we enclose the class member definitions in a namespace in this project.